airflow/configuration.py

```diff
@@ -567,8 +567,12 @@ def get_airflow_test_config(airflow_home):
         AIRFLOW_HOME = conf.get('core', 'airflow_home')
         warnings.warn(msg, category=DeprecationWarning)
 
-
-WEBSERVER_CONFIG = AIRFLOW_HOME + '/webserver_config.py'
+if 'WEBSERVER_CONFIG' in os.environ:
+    WEBSERVER_CONFIG = os.environ['WEBSERVER_CONFIG']
+elif conf.get('webserver', 'webserver_config'):
+    WEBSERVER_CONFIG = conf.get('webserver', 'webserver_config')
```

Comment:
'webserver', 'webserver_config' is a little bit... "duplicatey". How about 'webserver', 'config_file'?

That way it could be set via `AIRFLOW__WEBSERVER__CONFIG_FILE` environment variable automatically.

Also we can use the `fallback` arg so all of this just becomes:


```python
WEBSERVER_CONFIG=confi.get('webserver', 'config_file', fallback=AIRFLOW_HOME + '/webserver_config.py')
```